src/patternfly/components/Scrollspy/scrollspy.scss

```diff
@@ -0,0 +1,168 @@
+    --pf-c-scrollspy__link--focus--before--BorderTopWidth: var(--pf-c-scrollspy--m-vertical__link--focus--before--BorderTopWidth);
+    --pf-c-scrollspy__link--focus--before--BorderLeftWidth: var(--pf-c-scrollspy--m-vertical__link--focus--before--BorderLeftWidth);
+    --pf-c-scrollspy__item--m-current__link--before--BorderTopWidth: var(--pf-c-scrollspy--m-vertical__item--m-current__link--before--BorderTopWidth);
+    --pf-c-scrollspy__item--m-current__link--before--BorderLeftWidth: var(--pf-c-scrollspy--m-vertical__item--m-current__link--before--BorderLeftWidth);
+
+    .pf-c-scrollspy__list {
+      flex-direction: column;
+    }
+  }
+}
+
+.pf-c-scrollspy__list {
+  position: relative;
+  display: inline-flex;
+  padding-right: var(--pf-c-scrollspy__list--PaddingRight);
+  padding-left: var(--pf-c-scrollspy__list--PaddingLeft);
+  margin: 0;
+
+  &::before {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    content: "";
+    border-color: var(--pf-c-scrollspy__list--before--BorderColor);
+    border-style: solid;
+    border-width: var(--pf-c-scrollspy__list--before--BorderTopWidth) var(--pf-c-scrollspy__list--before--BorderRightWidth) var(--pf-c-scrollspy__list--before--BorderBottomWidth) var(--pf-c-scrollspy__list--before--BorderLeftWidth);
+  }
+}
+
+.pf-c-scrollspy__link {
+  position: relative;
+  padding-top: var(--pf-c-scrollspy__link--PaddingTop);
+  padding-right: var(--pf-c-scrollspy__link--PaddingRight);
+  padding-bottom: var(--pf-c-scrollspy__link--PaddingBottom);
+  padding-left: var(--pf-c-scrollspy__link--PaddingLeft);
+  border: 0;
+  outline-offset: var(--pf-c-scrollspy__link--OutlineOffset);
+
+  &:hover {
+    --pf-c-scrollspy__item-text--Color: var(--pf-c-scrollspy__link--hover__item-text--Color);
+  }
+
+  &:focus {
+    --pf-c-scrollspy__item-text--Color: var(--pf-c-scrollspy__link--focus__item-text--Color);
+    --pf-c-scrollspy__link--before--BorderTopWidth: var(--pf-c-scrollspy__link--focus--before--BorderTopWidth);
+    --pf-c-scrollspy__link--before--BorderLeftWidth: var(--pf-c-scrollspy__link--focus--before--BorderLeftWidth);
+    --pf-c-scrollspy__link--before--BorderColor: var(--pf-c-scrollspy__link--focus--before--BorderColor);
+  }
+
+  &::before {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    content: "";
+    border-color: var(--pf-c-scrollspy__link--before--BorderColor);
+    border-style: solid;
+    border-width: var(--pf-c-scrollspy__link--before--BorderTopWidth) var(--pf-c-scrollspy__link--before--BorderRightWidth) var(--pf-c-scrollspy__link--before--BorderBottomWidth) var(--pf-c-scrollspy__link--before--BorderLeftWidth);
+  }
+
+  + .pf-c-scrollspy__link {
+    --pf-c-scrollspy__link--PaddingLeft: var(--pf-c-scrollspy__link--link--PaddingLeft);
+  }
+}
+
+// stylelint-disable
+.pf-c-scrollspy__item {
+  &.pf-m-current {
+    display: flex;
```

Comment:
To increase clickable area, seems like it would be better to move `display: flex; to line [103](https://github.com/patternfly/patternfly/pull/3596/files#diff-32f68850123d67016f257f021955f786ca61f4bbe4e33f01ecd501e305399e2fR103) and set `.pf-c-scrollspy__link` to `flex-basis: 100%;` or `flex: 1 1 100%;`

![Screen Shot 2020-10-20 at 3 29 15 PM](https://user-images.githubusercontent.com/5385435/96635195-8bf98e80-12e9-11eb-8221-bb90134ef6ef.png)